cupy/cudnn.pyx

```diff
@@ -382,6 +382,13 @@ def set_dropout_descriptor(desc, handle, dropout):
     cudnn.setDropoutDescriptor(desc.value, handle, dropout, 0, 0, 0)
 
 
+def create_ctc_loss_descriptor(data_type):
```

Comment:
I want to hide `descriptor` and wrap cuDNN interface because raw cuDNN API is too complex.

This is high level API example.
https://github.com/cupy/cupy/blob/834f9f358971a65c82fdbdf3bed72ad60978d158/cupy/cudnn.pyx#L311

Cloud you remove this function from this PR?

Reply:
OK. Thank you for your comment.
Do you mean should I define the other high level API function like following?
```
def ctc_loss(data_type, other_args_for_cudnnCTCLoss):
    # create descriptor
    desc = Descriptor(cudnn.createCTCLossDescriptor(),
                    py_cudnn.destroyCTCLossDescriptor)
    cudnn.setCTCLossDescriptor(desc.value, data_type)

     # compute workspace
     getCTCLossWorkspaceSize(hoge, worksize)
     # allocate worksize
     workspace = ...
     # compute CTC loss
      CTCLoss(hoge, workspace, other_args_for_cudnnCTCLoss)
      return loss, gradients
```

Reply:
Yes, something like that.

Reply:
Cloud you rename this function or delete this?
```suggestion
 def _create_ctc_loss_descriptor(data_type):
```